components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2203,9 +2203,18 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
                 throw new Error(`No Stripe customer profile for '${attributionId}'`);
             }
 
-            await this.stripeService.setDefaultPaymentMethodForCustomer(customerId, setupIntentId);
-            await this.stripeService.createSubscriptionForCustomer(customerId, attributionId);
+            //TODO: DO NOT MERGE
+            await this.billingService.createStripeSubscription({
+                attributionId,
+                customerId,
```

Comment:
I added this customerId here, despite being able to look it up via attributionId, due to the race condition that seems to still persist while I was testing in the preview environment.